pandas/core/reshape.py

```diff
@@ -1213,7 +1216,7 @@ def make_axis_dummies(frame, axis='minor', transform=None):
         labels = cat.codes
         items = cat.categories
 
-    values = np.eye(len(items), dtype=float)
+    values = np.eye(len(items), dtype=np.uint8)
```

Comment:
@jreback this is a new change from what you saw earlier ( this is ``make_axis_dummies` in core/reshape.py.

One of the panel tests made use of it. I changed it for consistency with the new `get_dummies`, but happy to change it back to floats, and adjust the test.